stdlib/sql/source_validator.go

```diff
@@ -4,7 +4,7 @@ import (
 	neturl "net/url"
 	"strings"
 
-	"github.com/bonitoo-io/go-sql-bigquery"
+	bigquery "github.com/bonitoo-io/go-sql-bigquery"
```

Comment:
Did we need this alias? Or is it just a holdout from development?

Reply:
This import alias got added by the vscode go plugin, maybe to leverage this to bypass package name conflicts in the future.